src/main/java/laledevri/laladevri/App.java

```diff
@@ -8,6 +8,20 @@
 {
     public static void main( String[] args )
     {
+
+int target2 = -5;
+
+
+
+	int num3 = 3;
+
+
+
+final int deneme2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "deneme2" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)